packages/migrate/src/commands/DbPull.ts

```diff
@@ -480,6 +480,8 @@ ${`Run ${chalk.green(getCommandWithExecutor('prisma generate'))} to generate Pri
           message += warning.affected.map((it) => `- View "${it.view}", Field: "${it.field}"`).join('\n')
         } else if ([23, 24, 25].includes(warning.code)) {
           message += warning.affected.map((it) => `- View "${it.view}"`).join('\n')
+        } else if (warning.code === 28) {
+          message += warning.affected.map((it) => `- Model "${it.model}"`).join('\n')
```

Comment:
Couldn't this be another case for line 461?

Reply:
Ope, yeah, it seems to be :) I'll update that